apps/knockdog/src/features/map/ui/MapView.tsx

```
@@ -1,73 +1,95 @@
-import { NaverMap, Overlay } from '@knockdog/naver-map';
-import { ClusterMarker } from './ClusterMarker';
-import { PlaceMarker } from './PlaceMarker';
-import { useCurrentLocation } from '@shared/lib';
-import { CurrentLocationMarker } from '@shared/ui/naver-map/CurrentLocationMarker';
+import { Map, Marker } from '@knockdog/react-naver-map';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)